app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -235,7 +294,7 @@ export default class AMRHeatmapVis extends React.Component {
       // in the rightmost box, and vice versa.
       {
         rowLabels: rows,
-        columnLabels: columns,
+        columnLabels: columns, // needs to contain selected metadata *values*
```

Comment:
Can you change the comment to `Each columnLabel object contains the sample name, id, and all metadata values`? (I believe it's all metadata values, not selected?)

This part of the heatmap implementation is a little weird right now, so it's nice to have a thorough comment.